src/mono/System.Private.CoreLib/src/System/Reflection/Emit/RuntimeTypeBuilder.Mono.cs

```diff
@@ -34,7 +34,6 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-#if MONO_FEATURE_SRE
```

Comment:
What is `MONO_FEATURE_SRE` used for.[ I see it is defined for Mono,](https://github.com/dotnet/runtime/blob/70887088db9c0c4a1825f768efd84b323b0a055a/src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj#L120) but is it ever undefined?

Reply:
Not sure for what it's used, but [got a feedback](https://github.com/dotnet/runtime/pull/81518#discussion_r1094363517): `You could remove this code, we are not using this ifdef in dotnet/rutnime`

Reply:
It's a holdover from mono/mono.  There, instead of using the linker to remove SRE on platforms without dynamic code generation, we used compiler defines. 

It's probably safe to remove these ifdefs